### DISCUSSION TITLE
Please Add Save Button And Create Empty Activity

### DISCUSSION THREAD

**Discussion by getsettalk** [tyron12233/CodeAssist#196] [Ideas]
Hello sir, thank you so much for developing this app.💕

I  want to add some features in app,

1]. Add Button or Option for **_### Create Empty Activity file_** in  project .

2]. Please add custom **_save button for code_**, because sometime code don't save automatically

3]. Please add colorful text for code function variable etc.

![Screenshot_2022-02-11-12-32-55-83_2870cb4d7e72f8050b62d08052357b72](https://user-images.githubusercontent.com/49394996/153549706-dd2e6d49-c8d6-4c95-aa18-2c0608df434b.jpg)

In this **screenshot you can view not showing** options to create new Activity or Empty Activity. 

@MikeAndrson

**Comment by Rystars** [comment]
For number 2, don't, it's better to be automatic
For number 3 you can customize your own color in version 0.2.6

**Reply by getsettalk** [reply]
For number 2, i know that is better , but  sometimes code doesn't saved automatic. And Sometimes layout changea not saved in XML file.

**Reply by Rystars** [reply]
That means autocomplete must be improved again so that it always saves

**Comment by getsettalk** [comment]
Yeah 👍🏻
